components/button/button-styles.js

```diff
@@ -25,4 +25,9 @@ export const buttonStyles = css`
 		outline: 2px solid var(--d2l-button-focus-color, var(--d2l-color-celestine));
 		outline-offset: var(--d2l-button-focus-offset, 2px);
 	}
+	@media (prefers-contrast: more) {
```

Comment:
Does setting this affect/control the value for this media-query?
![image](https://github.com/user-attachments/assets/ec3a6625-ad87-4c7b-9fd9-64c945730310)

Reply:
So it does cause the media query to trigger, yes. But visually it has no impact other than making every button 4px wider since we're just setting a transparent border.